TelemetrySystem/src/main/java/tddmicroexercises/telemetrysystem/implementation/ClientConnection.java

```diff
@@ -0,0 +1,37 @@
+package tddmicroexercises.telemetrysystem.implementation;
+
+import tddmicroexercises.telemetrysystem.IClientConnection;
+
+import java.util.Random;
+
+public class ClientConnection implements IClientConnection {
```

Comment:
Lets probably name this class to TelemetryClient Connection

Reply:
addressed the review comment